cache-extension-demo/README.md

```diff
@@ -74,16 +107,6 @@ aws dynamodb put-item \
 ```
 > Note: Dynamodb values are stored under the key generated using the format `<table><hyphen><hashkey><hyphen><rangekey>`
 
-## Compile package and dependencies
```

Comment:
Is this really no longer necessary? How would someone get the compiled Go binary?

Reply:
I see that it was moved.

Reply:
Moved it to the top section before Deployment steps